eth2deposit/intl/zh-CN/cli/generate_keys.json

```
@@ -0,0 +1,35 @@
+{
+    "validate_password": {
+        "msg_password_prompt": "输入密码以保护您的验证者密钥库",
+        "msg_password_confirm": "重复以确认",
```

Note for myself:

```suggestion
        "msg_password_confirm": "重复输入以确认",
```

I can create crowdin issue later.

---

We can merge here and then I go propose the change on Crowdin.
@wackerow what do you think?